docker-compose.yml

```diff
@@ -49,6 +49,8 @@ services:
     container_name: "algorand-sandbox-postgres"
     ports:
       - 5433:5432
+    # https://stackoverflow.com/questions/28311825/root-execution-of-the-postgresql-server-is-not-permitted
```

Comment:
I appreciate the reference and this fix. However, I feel we should document the fix within the PR description rather than inline as a comment.

Reply:
Thanks for the feedback, I have made the change accordingly.